app/views/govuk_publishing_components/components/_success_alert.html.erb

```diff
@@ -1,4 +1,7 @@
-<% description ||= nil %>
+<%
+  description ||= nil
+  id = "govuk-notification-banner-title-#{SecureRandom.hex(4)}"
```

Comment:
```suggestion
  id ||= "govuk-notification-banner-title-#{SecureRandom.hex(4)}"
```
we can allow custom ids as well. more importantly, we'd need `labelledby` on [L10](https://github.com/alphagov/govuk_publishing_components/pull/2004/files#diff-81b823e894a91648587dfbb3e5a12dfb1046735370fe56f4286dc39a96b1b39aR10) to use this `id`

Reply:
So, lemme clarify the 2 asks here. The first is to add an `id` option to the component API. Very doable. The 2nd I'm not quite clear on. Where does the `aria-labelledby` need to go and what should it be? Am I completely misunderstanding?

Reply:
I could've been more clear. The point of having an `id` on the 'title' element is for [`aria-labelledby` to point to it](https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_success_alert.html.erb#L7). If we're going to make this `id` dynamic, we'd need to update the `aria-labelledby` reference to point to this custom `id`.

Reply:
we may want to call it `title_id` to reflect where it will be applied in the API – we tend to use `id` when the attribute is applied to the top-level element/component-level

Reply:
Ahaaaaaaa gotcha. Thanks Alex. Will action.